crates/wasmprinter/tests/all.rs

```diff
@@ -46,3 +47,29 @@ fn offsets_and_lines_smoke_test() {
 
     assert_eq!(actual, expected);
 }
+
+#[test]
+fn custom_indent_text() {
```

Comment:
Nowadays I try to avoid adding tests here since they're difficult to update, so mind moving this test to `tests/cli/*.wat` whch uses the `--indent-text` argument?